### PR TITLE
Initialize setup.py

### LINKED ISSUE #3: Create setup.py
Create setup script using [setup.py from sandpiper](https://github.com/pacificclimate/sandpiper/blob/master/setup.py) as a template.

### PR DESCRIPTION
**Pull request by eyvorchuk**
This PR solves [issue 3](https://github.com/pacificclimate/wps-tools/issues/3), which creates a `setup.py` script. This is based on the script from `sandpiper`, and is thus subject to modifications as needed.

### CODE REVIEW

**Review by nikola-rados — CHANGES_REQUESTED** [review]
I think this is _fine_, but I would prefer to base it off of something we created rather than one of the bird items.  I think the [`pdp_util`](https://github.com/pacificclimate/pdp_util/blob/master/setup.py) or [`crmprtd`](https://github.com/pacificclimate/crmprtd/blob/master/setup.py) are better starting points for us to work off of.

**Review by sum1lim — COMMENTED** [review]
To the best of my short knowledge of building `setup.py`, everything looks fine to me.

File: wps_tools/__version__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+
+# This information is located in its own file so that it can be loaded
+# without importing the main package when its dependencies are not installed.
+# See: https://packaging.python.org/guides/single-sourcing-package-version
+
+__author__ = """Eric Yvorchuk"""
+__email__ = 'eyvorchuk@uvic.ca'
+__version__ = '0.1.0'
```
Comment (nikola-rados) [inline wps_tools/__version__.py]: I don't think this file is necessary. We can define these things in the `setup.py`

File: wps_tools/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+"""Top-level package for wps-tools."""
+
+from .__version__ import __author__, __email__, __version__  # noqa: F401
```
Comment (nikola-rados) [inline wps_tools/__init__.py]: This will probably be changed with the comment below.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    description="A collection of modules used to help create wps processes",
+    keywords="wps pywps wps_tools",
+    version='.'.join(str(d) for d in __version__),
+    url="https://github.com/pacificclimate/wps-tools",
+    author="Eric Yvorchuk",
+    author_email="eyvorchuk@uvic.ca",
+    packages=find_packages(),
+    package_data={
+        'wps_tools': ["wps_tools/*.py"],
+    },
```
Comment (nikola-rados) [inline setup.py]: I believe you are mistaking this for the `packages` portion.  This is for anything data-wise needed to run the package. Have a look [here](https://docs.python.org/3/distutils/setupscript.html#installing-package-data) for more info.

**Review by nikola-rados — CHANGES_REQUESTED** [review]
This is better than before, much more simple. We can still improve in a few places.

File: wps_tools/__init__.py (hunk 2 above)
Comment (eyvorchuk) [reply wps_tools/__init__.py]: Since this is the top level, how do I properly import these variables from `setup.py` even when it's one directory above?

Comment (nikola-rados) [reply wps_tools/__init__.py]: What do you need to import these for?

Comment (eyvorchuk) [reply wps_tools/__init__.py]: I was just going off what's done in `sandpiper`. Does this `__init__` file need anything, then?

Comment (nikola-rados) [reply wps_tools/__init__.py]: Ya we can just leave it empty!

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import sys
+
+from setuptools import setup
+
+__author__ = "Eric Yvorchuk"
+__email__ = "eyvorchuk@uvic.ca"
```
Comment (nikola-rados) [inline setup.py]: You can just directly set these values below.

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+__version__ = "0.1.0"
```
Comment (nikola-rados) [inline setup.py]: I prefer the way we were handling the version before.

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+setup(
+    name="wps_tools",
+    description="A collection of modules used to help create wps processes",
+    keywords="wps pywps wps_tools",
+    version=__version__,
+    url="https://github.com/pacificclimate/wps-tools",
+    author=__author__,
+    author_email=__email__,
+    packages=["wps_tools"],
```
Comment (nikola-rados) [inline setup.py:15]: I _think_ it is better to use `find_packages()` but I don't know for sure. So we can leave this as is for now.

File: wps_tools/__init__.py [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+"""Top-level package for wps-tools."""
```
Comment (nikola-rados) [inline wps_tools/__init__.py]: This can be completely empty.

**Review by nikola-rados — CHANGES_REQUESTED** [review]
Just a few more minor  things and we're good to go here.

**Pull request merged**